src/main/java/com/notify/notify/service/SMSCRUDService.java

```diff
@@ -0,0 +1,60 @@
+package com.notify.notify.service;
+
+import com.notify.notify.exceptions.InValidRequestException;
+import com.notify.notify.exceptions.NotFoundException;
+import com.notify.notify.model.SMS;
+import com.notify.notify.model.request.SendSMSRequest;
+import com.notify.notify.repository.SMSRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+
+@Service
+public class SMSCRUDService {
+
+    @Autowired
+    private SMSRepository smsRepository;
+
+    public Long saveSMS(SendSMSRequest sendSmsRequest)
+    {
+        if(sendSmsRequest.getPhoneNumber()==null||sendSmsRequest.getPhoneNumber().length()!=10)
+            throw new InValidRequestException("Invalid phone number");
+        SMS sms=new SMS();
+        sms.setPhoneNumber(sendSmsRequest.getPhoneNumber());
+        sms.setMessage(sendSmsRequest.getMessage());
+        sms.setStatus("accepted");
+        sms.setCreatedAt(Timestamp.from(Instant.now()));
+        sms.setUpdatedAt(Timestamp.from(Instant.now()));
+        sms=smsRepository.save(sms);
+        return sms.getId();
+    }
+
+    public SMS getSMSById(Long smsId)
+    {
+        if(smsId==null)throw new InValidRequestException("Invalid request_id");
```

Comment:
Use ObjectUtils.isEmpty(smsId) instead of having null check.
You can have one ErrorMessageConstant class to have all constants there. (Use constant instead of "Invalid request_id"